src/bvals/boundary_conditions.hpp

```diff
@@ -43,6 +42,12 @@ TaskStatus ApplyBoundaryConditionsMD(std::shared_ptr<MeshData<Real>> &pmd);
 TaskStatus ApplyBoundaryConditionsOnCoarseOrFineMD(std::shared_ptr<MeshData<Real>> &pmd,
                                                    bool coarse);
 
+inline TaskStatus ApplyBoundaryConditions(std::shared_ptr<MeshBlockData<Real>> &rc);
```

Comment:
Why is this declared again here when it is defined a few lines above? I must be missing something.

Reply:
Not sure how that happened, thanks for noticing this, removed.